### PR TITLE
Feature: add request for streaming output of execution

### PR DESCRIPTION
**Pull request by madcampos**
Add request to status update command so it can be used for the spinner.

https://github.com/user-attachments/assets/3288e2ad-1214-4066-b4f7-fa3f46d46aed

### CODE REVIEW

File: src/ui/src/common/extension_command.ts [hunk 1]
```diff
@@ -74,3 +74,20 @@ export declare interface AdapterExecuteCommand extends ExtensionCommand {
...
+/** Adapter's "status check" command. */
+export declare interface AdapterStatusCheckCommand extends ExtensionCommand {
+  cmdId: 'status_check';
+  modelPath: string;
+  settings: Record<string, any>;
+  deleteAfterConversion: boolean;
+}
+
+/** Adapter's "status check" command response. */
+export declare interface AdapterStatusCheckResponse extends ExtensionResponse {
```
[the remark this answers is not in the document]
Comment (madcampos) [reply src/ui/src/common/extension_command.ts:87]: Yes, absolutely, I'm adding that next.

**Review by vprajapati-tt — APPROVED** [review]
Looks good to me! Appreciate the framework for adding Perf Traces as well, hopefully we can add them in soon!

**Comment by vprajapati-tt** [comment]
I did want to create an issue BTW. We should try and implement `Squash and Merge` instead of just merge for the PRs. It helps with providing `atomic` changes to the repo "per-feature". It's similar to how we maintain `tt-mlir` and `tt-forge-fe`.

**Comment by madcampos** [comment]
Oh, right, I didn't knew we were squashing commits. Will do that from now on

**Pull request merged**